ui/src/app/pages/demographic-survey.tsx

```diff
@@ -0,0 +1,131 @@
+import * as React from 'react';
+import { useEffect, useState } from 'react';
+import * as fp from 'lodash/fp';
+
+import { Button } from 'app/components/buttons';
+import DemographicSurveyV2 from 'app/components/demographic-survey-v2';
+import { TooltipTrigger } from 'app/components/popups';
+import { WithSpinnerOverlayProps } from 'app/components/with-spinner-overlay';
+import { profileApi } from 'app/services/swagger-fetch-clients';
+import { useNavigation } from 'app/utils/navigation';
+import { profileStore } from 'app/utils/stores';
+
+interface DemographicSurveyProps extends WithSpinnerOverlayProps {
+  returnAddress: string;
+}
+
+export const DemographicSurvey = (props: DemographicSurveyProps) => {
+  const [errors, setErrors] = useState(null);
+  const [changed, setChanged] = useState(false);
+  const [initialSurvey, setInitialSurvey] = useState(null);
+  const [profile, setProfile] = useState(null);
+  const [loading, setLoading] = useState(true);
+  const [, navigateByUrl] = useNavigation();
+
+  const { hideSpinner, showSpinner, returnAddress } = props;
```

Comment:
Now that I look at it, it is not. I removed the property. It will now just return to the profile page until we have a reason to return somewhere else. Thanks for catching that.